core/controllers/feedback_test.py

```diff
@@ -104,7 +104,7 @@ def test_non_logged_in_users_cannot_create_threads_and_messages(self):
             'text': self.UNICODE_TEST_STRING,
         }, self.csrf_token, expect_errors=True, expected_status_int=401)
 
-        thread_url = '%s/%s/%s' % (
+        thread_url = '%s/%s.%s' % (
```

Comment:
ditto, why not '%s/%s' and remove ref to exp_id

Reply:
Done